samples/monitoring_schedule/README.md

```diff
@@ -0,0 +1,36 @@
+# Bring Your Own Monitoring Schedule Sample
+
+This sample demonstrates how to start Monitoring scheduling using your own Monitor scheduling script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.   
+
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](To do). It also assumes you have a job of one of these types already created `DataQuality|ModelQuality|ModelBias|ModelExplainability`.
```

Comment:
link to the directories for job definitions and change the wording to say job definitions